### ISSUE TITLE
Sen2agriPlatformInstallAndConfigCore.sh is buggy

### ISSUE THREAD

**Issue by savmickael** [Sen2Agri/Sen2Agri-System#9]
Sen2Agri-System/install_platform/sen2agriPlatformInstallAndConfigCore.sh refers to the version 1.4. Therefore the script cannot install the rpm file for processors which is named with 1.6.
Change the version number fix the issue.

**Comment by lnicola** [comment]
Fixed (but not pushed to GitHub!) in commit `7d88d8f9ca7778954077c3cffc89865fd41a6b7e`.

The file currently looks like this. Please see if it works for you.

```bash
#!/bin/bash
#set -x #echo on

##
## SCRIPT: INSTALL AND CONFIGURE PLATFORM SEN2AGRI CORE
##
##
## SCRIPT STEPS
##     - INSTALL OTB, SEN2AGRI PROCESSORS
################################################################################################
## SCRIPT USAGE:
##
## open a terminal go into delivery install_script folder:
## cd /path/to/install_script
## sudo ./sen2agriPlatormInstallAndConfigCore.sh
################################################################################################
: ${SYS_ACC_NAME:="sen2agri-service"}
#-----------------------------------------------------------------------------------------#
function create_system_account()
{
   #create system account for running services
   adduser -m ${SYS_ACC_NAME}
}
#-----------------------------------------------------------#
function install_RPMs()
{
   ##########################################################
   ####  OTB, SEN2AGRI-PROCESSORS, SEN2AGRI-SERVICES
   ##########################################################
   ##install EPEL for packages dependencies installation
   yum -y install epel-release
   yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-7.3-x86_64/pgdg-centos94-9.4-3.noarch.rpm

   ##install Orfeo ToolBox
   yum -y install ../rpm_binaries/otb-*.rpm

   ##install Sen2Agri Processors
   yum -y install ../rpm_binaries/sen2agri-processors-*.centos7.x86_64.rpm

   ln -s /usr/lib64/libproj.so.0 /usr/lib64/libproj.so
   ldconfig
}
###########################################################
##### MAIN                                              ###
###########################################################

#-----------------------------------------------------------#
####  OTB, SEN2AGRI PROCESsORS INSTALL  & CONFIG ######
#-----------------------------------------------------------#
## install binaries
install_RPMs

## create system account
create_system_account
```

**Comment by savmickael** [comment]
ok seems nice. I will tested with next released version.
Thanks
